app/api/v1/users/views.py

```diff
@@ -42,3 +43,27 @@ def post(self):
             return {"status": 201, "data": [{"id": user["id"], "message": "User Created"}]}, 201
         else:
             return {"status": 400, "error": "Password and Password confirmation do not match. Please try again."}, 400
+
+
+class UserLogin(BaseUsers):
+    def post(self):
+        parser = reqparse.RequestParser()
+        parser.add_argument('username', type=str, location="json", help = 'Username cannot be blank', required = True)
+        parser.add_argument('password', type=str, location="json", help = 'Password cannot be blank', required = True)
+
+        args = parser.parse_args()
+
+        data = request.get_json()
+        user = {
+            "username": data["username"],
+            "password": data["password"]
+        }
+        user_to_log_in = self.users.get_specific(user["username"])
+        if user_to_log_in:
+            if user_to_log_in["password"] == user["password"]:
+                access_token = create_access_token(user["username"], expires_delta=datetime.timedelta(days=365))
+                return {"status": 200, "data": [{"id": user_to_log_in["id"], "access_token": access_token, "message": "User Logged In"}]}
```

Comment:
i think 365 days should be token expiry for a developer, not a normal user?

Reply:
Yea I forgot to remove that while I was testing the API on postman. I'll shorten it.